src/singletons/Paths.cpp

```diff
@@ -37,7 +37,7 @@ bool Paths::isPortable()
 
 QString Paths::cacheDirectory()
 {
-    static QStringSetting cachePathSetting = [] {
```

Comment:
The lambda is immediately invoked so the name `getCachePathSetting` doesn't make much sense.